src/Stratis.Features.FederatedPeg/FederatedPegSettings.cs

```diff
@@ -50,6 +50,16 @@ public sealed class FederatedPegSettings : IFederatedPegSettings
         /// </summary>
         public static readonly Money InputTransactionFee = Money.Coins(0.0001m);
 
+        /// <summary>
+        /// Fee applied to consolidating transactions. Roughly the same as the fee for withdrawals.
+        /// </summary>
+        public static readonly Money ConsolidationFee = Money.Coins(0.005m);
+
+        /// <summary>
+        /// The maximum number of inputs we want our built withdrawal transactions to have. We don't want them to get too big for Standardness reasons.
```

Comment:
`too big for Standardness reasons.`

There is no standard tx policy about tx size, only a consensus rule that ensures that tx is not bigger then the block size - header size. So as long as it fits into a block it's totally fine. 

So I'd bump this constant to 200 or smth.

Reply:
There is a standardness policy about transaction size, it's 100kb.

Here it is for bitcoin: https://github.com/bitcoin/bitcoin/blob/3c098a8aa0780009c11b66b1a5d488a928629ebf/src/policy/policy.h#L24

Here it is for us:
https://github.com/stratisproject/StratisBitcoinFullNode/blob/745004194c0024f95e306061fd82aab8e4a7bdc3/src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs#L634

We could still go higher but I think it's more problematic - bigger transactions to pass around and sign, I think progress would be quite a bit slower.

Reply:
Ah, right.